templates/treat_ad.yaml

```diff
@@ -0,0 +1,61 @@
+Structural_Biology_Core_Folder: &struct_bio_folder
```

Comment:
Am I correct in assuming that this YAML object would be ignored by SynapseFormation because it isn't of type `Project` despite being at the top level? Or will SynapseFormation have to leverage the fact that it has an anchor to make that distinction (_i.e._ whether it needs to be created or not)?

Reply:
That's a good question worth looking into.  I have a feeling it will attempt to parse and convert to JSON - which then could be accessed by the index of the object.